<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1910</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(1, 37, 55);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="MainBar" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1861</width>
      <height>1011</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="SideTab" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>281</width>
       <height>961</height>
      </rect>
     </property>
     <widget class="QFrame" name="SideTabFrame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>901</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(33, 158, 188);
border-top-left-radius: 32px;
border-bottom-left-radius: 32px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Lessons">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>372</y>
         <width>279</width>
         <height>42</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>26</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Lessons</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../linear/note-2.svg</normaloff>../linear/note-2.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>350</y>
         <width>271</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(142, 202, 230);
border-top-left-radius: 0px;
border-bottom-left-radius: 0px;
border-top-right-radius: 32px;
border-bottom-right-radius: 32px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>770</y>
         <width>281</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="Settings">
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>26</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../linear/setting-3.svg</normaloff>../linear/setting-3.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>20</y>
         <width>241</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="Kaway">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>36</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>Kaway</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Dot">
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>36</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 170, 0);</string>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="Profile">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>480</y>
         <width>279</width>
         <height>42</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>26</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 255, 255)</string>
       </property>
       <property name="text">
        <string>Profile</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../linear/user-square.svg</normaloff>../linear/user-square.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="Home">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>270</y>
         <width>279</width>
         <height>42</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>26</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0)</string>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../linear/element-3.png</normaloff>../linear/element-3.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <zorder>frame</zorder>
      <zorder>Lessons</zorder>
      <zorder>verticalLayoutWidget</zorder>
      <zorder>horizontalLayoutWidget</zorder>
      <zorder>Profile</zorder>
      <zorder>Home</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="RightContainer" native="true">
     <property name="geometry">
      <rect>
       <x>279</x>
       <y>-1</y>
       <width>1581</width>
       <height>1011</height>
      </rect>
     </property>
     <widget class="QFrame" name="LowerFrame">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>1501</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(82, 126, 147);
border-radius: 32px</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="MidFrame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>1521</width>
        <height>571</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(103, 153, 176);
border-radius: 32px</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="AnswerText">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>500</y>
         <width>751</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>36</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>A</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="UpperFrame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>1541</width>
        <height>541</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(142, 202, 230);
border-radius: 32px
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="CameraFrame">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>20</y>
         <width>991</width>
         <height>491</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="StartCamera">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>131</width>
         <height>211</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>211</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>28</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(82, 126, 147);
border-radius: 32px
</string>
       </property>
       <property name="text">
        <string>CAM</string>
       </property>
      </widget>
      <widget class="QPushButton" name="StartDetection">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>131</width>
         <height>211</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>131</width>
         <height>211</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>28</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(82, 126, 147);
border-radius: 32px
</string>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
      <widget class="QFrame" name="Error">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>110</y>
         <width>941</width>
         <height>331</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(103, 153, 176);
border-radius: 32px</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="AnswerText_3">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>500</y>
          <width>751</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Inter</family>
          <pointsize>36</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>A</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
       <widget class="QLabel" name="ErrorText">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>911</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Inter</family>
          <pointsize>32</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Error:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>790</y>
        <width>1541</width>
        <height>101</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1541</width>
        <height>101</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(142, 202, 230);
border-radius: 32px
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="NextModule">
       <property name="geometry">
        <rect>
         <x>1260</x>
         <y>10</y>
         <width>271</width>
         <height>81</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>271</width>
         <height>81</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>28</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(82, 126, 147);
border-radius: 32px
</string>
       </property>
       <property name="text">
        <string>Complete</string>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>0</y>
         <width>831</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="UserAnswer">
          <property name="maximumSize">
           <size>
            <width>829</width>
            <height>61</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>48</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 255, 0);</string>
          </property>
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ReviewButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>339</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Click here to review again</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="Check">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>121</width>
         <height>101</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>101</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../linear/check.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="mainframe">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1581</width>
        <height>901</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(3, 72, 106);
border-top-right-radius: 32px;
border-bottom-right-radius: 32px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>1061</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="Module">
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>32</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Module 1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="subtopic">
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>26</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Subtopic 1: Letters</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="NotifButton">
       <property name="geometry">
        <rect>
         <x>1460</x>
         <y>40</y>
         <width>61</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(33, 158, 188);
    border-radius: 30px;
    }
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../AppData/Local/Programs/Python/Python311/Lib/site-packages/qt6_applications/Qt/bin/notification-bing.svg</normaloff>../../../../AppData/Local/Programs/Python/Python311/Lib/site-packages/qt6_applications/Qt/bin/notification-bing.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="NotifNum">
       <property name="geometry">
        <rect>
         <x>1510</x>
         <y>30</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Inter</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(251, 133, 0);
    border-radius: 20px;
  </string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <zorder>mainframe</zorder>
     <zorder>LowerFrame</zorder>
     <zorder>MidFrame</zorder>
     <zorder>UpperFrame</zorder>
     <zorder>frame_4</zorder>
    </widget>
    <widget class="QFrame" name="MainFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1861</width>
       <height>961</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(2, 48, 71);
border-radius: 32px
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>MainFrame</zorder>
    <zorder>RightContainer</zorder>
    <zorder>SideTab</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
